web/components/FcPaneMapPopup.vue

```diff
@@ -295,7 +300,8 @@ export default {
         if (this.featureLocationsEditIndex !== -1) {
           return false;
         }
-        return this.locationEditIndex === -1 && this.locationsEdit.length >= MAX_LOCATIONS;
+        return this.locationsEditIndex === -1
+          && this.locationsEditSelection.locations.length >= MAX_LOCATIONS;
```

Comment:
can we move this condition (the `length >= MAX_LOCATIONS` one) to a getter, e.g. `locationsEditFull`?  we can then use it here and in `FcPaneMapPopup`.